repo/packages/K/kibana/800/resource.json

```diff
@@ -0,0 +1,19 @@
+{
+  "assets": {
+    "uris": {
+      "kibana-tar-gz": "https://downloads.mesosphere.com/elastic/assets/kibana-7.3.2-linux-x86_64.tar.gz",
+      "init-sh": "https://downloads.mesosphere.com/kibana/assets/2.8.0-7.3.2/init.sh",
+      "nginx-conf-tmpl": "https://downloads.mesosphere.com/kibana/assets/2.8.0-7.3.2/nginx.conf.tmpl"
+    },
+    "container": {
+      "docker": {
+        "nginx": "nginx:1.15.12"
+      }
+    }
+  },
+  "images": {
+    "icon-small": "https://downloads.mesosphere.com/elastic/assets/icon-kibana-bb.svg",
+    "icon-medium": "https://downloads.mesosphere.com/elastic/assets/icon-kibana-bb.svg",
+    "icon-large": "https://downloads.mesosphere.com/elastic/assets/icon-kibana-bb.svg"
```

Comment:
FYI appending `@2x` to the file name makes it show up better on retina-display. Look [at the Note here](https://github.com/mesosphere/universe#images)

Reply:
Hm, we didn't upload an SVG with `@2x` appended to the name. SVGs should look nice on any resolution though, right? Differently from JPEGs and PNGs.